apb_devel/writing/reference.adoc

```diff
@@ -684,8 +686,9 @@ xref:../../apb_devel/cli_tooling.adoc#apb-devel-cli-install-prereqs-access-permi
 `apb list` command. If you do not have the `apb` tooling installed, you can use
 the route gathered from the previous step and run:
 +
+[subs=attributes+]
 ----
-$ curl -H "Authorization: Bearer $(oc whoami -t)" -k https://asb-1338-ansible-service-broker.172.17.0.1.nip.io/ansible-service-broker/v2/catalog
+$ curl -H "Authorization: Bearer $(oc whoami -t)" -k https://asb-1338-{oab-ns}.172.17.0.1.nip.io/ansible-service-broker/v2/catalog
```

Comment:
The 2 instances of `/ansible-service-broker/v2/catalog` should stay as-is for both upstream and downstream though?

Reply:
Turns out that you can get this value by simply curling the route, this is from my development environment:

```
$ curl -H "Authorization: Bearer $(oc whoami -t)" -k https://broker-automation-broker.172.17.0.1.nip.io/
{
  "paths": [
    "/apis",
    "/automation-broker/",
    "/healthz",
    "/healthz/ping",
    "/healthz/poststarthook/generic-apiserver-start-informers",
    "/metrics"
  ]
}
```

I would expect for `curl -H "Authorization: Bearer $(oc whoami -t)" -k https://asb-1338-{oab-ns}.172.17.0.1.nip.io/` to look something like:

```
{
  "paths": [
    "/apis",
    "/ansible-service-broker/",
    "/healthz",
    "/healthz/ping",
    "/healthz/poststarthook/generic-apiserver-start-informers",
    "/metrics"
  ]
}
```

After 3.11 we intend to move this to a more generic value (ie. `/osb/v2/catalog`) so that we can standardize that value for all deployments upstream and downstream. However, you should be able to reliably get the path to bootstrap/catalog by first getting the list of supported paths.